main.py

```diff
@@ -1,25 +1,24 @@
 ### Assignment: Regex and Files
-##Name:
-##Date: 
+##Name: Clarissa
+##Date: Beamish
 
 #don't forget to import regex
-
+import re
 
 ##(/5) Task 1: MODIFY the code below.
 #A# Change the regex from .* to what is required to capture an email address
 #B# Add a condition so that no blanks are printed.
 #C# Count the number of email addresses found and print a final output line.
 
 def reEmail(fname):
-  
+  count = 0
   fhand = open(fname,'r')
   for line in fhand:
-    extr = re.findall('.*',line)
-    if len(extr): 
+    extr = re.findall('.*@.*', line)
```

Comment:
Your regex will extract ALL the characters to the left and right of the @ symbol.
`extr = re.findall('\S+@\S+',line)`